lib/IRGen/GenDecl.cpp

```diff
@@ -1659,10 +1659,14 @@ getIRLinkage(const UniversalLinkageInfo &info, SILLinkage linkage,
 
   case SILLinkage::HiddenExternal:
   case SILLinkage::PrivateExternal:
-    return std::make_tuple(isDefinition
-                               ? llvm::GlobalValue::AvailableExternallyLinkage
-                               : llvm::GlobalValue::ExternalLinkage,
-                           llvm::GlobalValue::HiddenVisibility,
+    if (isDefinition) {
+      return std::make_tuple(llvm::GlobalValue::AvailableExternallyLinkage,
+                             llvm::GlobalValue::HiddenVisibility,
+                             llvm::GlobalValue::DefaultStorageClass);
+    }
+
+    return std::make_tuple(llvm::GlobalValue::ExternalLinkage,
```

Comment:
I'm confused.  Why is the declaration marked as being visible externally?

Reply:
It's importing a definition from another module (with DLLImport on Windows). How can it not be visible externally? The ExternalLinkage is not a change in my PR. I only changed this case to not have hidden visibility.

Reply:
Ah, for some reason, it seemed like it was changed here.